src/components/AnchorBase.tsx

```diff
@@ -35,13 +45,6 @@ export const AnchorBase = (props: Props) => {
       dispatch(connectAnchorBase(parentDoc, anchor, el));
     }
   }, []);
-  useEffect(() => {
-    if (!ref) return () => null;
-    return () => {
-      const parentDoc = getDoc(ref);
-      return dispatch(disconnectAnchor(parentDoc, ref));
-    };
-  }, [ref]);
```

Comment:
There were two independent UseEffects to trigger the disconnect here, which is double trouble in terms of the number of action issues on unmount. @rowanc1 does the change to a single still capture the cases covered by the original pair?

Reply:
Yeah - I think you adding the above seems to work. Did you try it?

Reply:
yes, various times :p